### PR TITLE
Update readme.txt

### PR DESCRIPTION
**Pull request by magicemilymiddleton**
PHP version update for plugin page when users look at plugin description in WordPress

## Description

Description of PHP version changed from 7.2 to 7.4
Fixes #

## Screenshots 
Screen Shot 2022-07-18 at 11 36 27 PM

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -414,7 +414,7 @@ Install the free LifterLMS plugin on your website from here on WordPress, then .
 
 LifterLMS Requires
 
-+ PHP 7.2 or later
++ PHP 7.4 or later
 + MySQL 5.6 or later
 + WordPress 4.0 or later
```
Comment (eri-trabiccolo) [inline readme.txt:419]: we also require WP 5.6 or later, can you change that too?

**Review by thomasplevy — CHANGES_REQUESTED** [review]
readme.txt is a generated file, it's contents are found here: https://github.com/gocodebox/lifterlms/tree/trunk/.wordpress-org/readme

generation is documented at https://github.com/gocodebox/lifterlms/tree/trunk/packages/dev#readme

**Comment by thomasplevy** [comment]
Also @willmiddleton-lifterlms the area in the screenshot is pulled from the lifterlms.com REST api -- updating the readme here won't update what you've highlighted in the screenshot

that needs to be updated on the WC product on LifterLMS.com

**Comment by magicemilymiddleton** [comment]
Okay, good to know, wasn't sure where that came from but found that in the readme file so just suggested anyway

**Comment by thomasplevy** [comment]
Closing in favor of merge codes implemented via: https://github.com/gocodebox/lifterlms/commit/e46388f562c6857d489373630a2c3b9af72f40d1

@willmiddleton-lifterlms the actual API returned value still needs to be updated at lifterlms.com if you haven't already done so

**Pull request closed without merging**